runtimes/pythonrt/worker_grpc_handler.go

```diff
@@ -184,61 +185,63 @@ func (s *workerGRPCHandler) Activity(ctx context.Context, req *userCode.Activity
 
 // ak functions
 
-func makeCallbackMessage(args []sdktypes.Value, kwargs map[string]sdktypes.Value) *callbackMessage {
-	callbackChan := make(chan sdktypes.Value, 1)
-	errorChannel := make(chan error, 1)
+func (w *workerGRPCHandler) callback(ctx context.Context, rid string, name string, fn func(context.Context, *sdkservices.RunCallbacks, sdktypes.RunID) (any, error)) (*callbackResponse, error) {
+	l := w.log.With(zap.String("runner", rid), zap.String("name", name))
+
+	startedAt := time.Now()
+
+	w.mu.Lock()
+	runner, ok := w.runnerIDsToRuntime[rid]
+	w.mu.Unlock()
+	if !ok {
+		l.Error("unknown runner ID", zap.String("id", rid))
+		return nil, errors.New("unknown runner ID")
+	}
+
+	l.Debug("sending callback")
 
 	msg := &callbackMessage{
-		args:           args,
-		kwargs:         kwargs,
-		successChannel: callbackChan,
-		errorChannel:   errorChannel,
+		name: name,
+		fn:   fn,
+		ch:   make(chan callbackResponse, 1),
+	}
+
+	runner.channels.callback <- msg
+
+	l.Debug("callback sent", zap.Duration("duration", time.Since(startedAt)))
```

Comment:
```suggestion
	l.Debug("callback sent", zap.Duration("duration", time.Since(startedAt)), zap.String("name", name))
```

Reply:
it's already at line 189, autoattached to all logs in this function.